.devcontainer/devcontainer.json

```diff
@@ -1,14 +1,26 @@
 {
   "name": "AWS CDK & Python Development Environment",
   "image": "mcr.microsoft.com/devcontainers/base:ubuntu-22.04",
+  "containerEnv": {
+    "ENV": "${localEnv:ENV}",
+    "SECRETS": "${localEnv:SECRETS}",
+    "AWS_ACCESS_KEY_ID": "${localEnv:AWS_ACCESS_KEY_ID}",
+    "AWS_SECRET_ACCESS_KEY": "${localEnv:AWS_SECRET_ACCESS_KEY}",
+    "AWS_SESSION_TOKEN": "${localEnv:AWS_SESSION_TOKEN}",
```

Comment:
We don't use these VARs in the CI/CD workflow.  When I do deployments from my laptop i set the `AWS_PROFILE` var.  Even the current doc says to use an aws profile https://github.com/Sage-Bionetworks-IT/openchallenges/blob/dev/README.md#deploy

Reply:
My understanding - or ChatGPT's :-) - is that the GitHub Action `aws-actions/configure-aws-credentials@v2` creates these `AWS_*` environment variables. By defining these env vars in `devcontainer.json`, we will be able to run the command `cdk deploy --all --concurrency 5 --require-approval never` inside the dev container.